### ISSUE TITLE
Don't rebuild docker on every config change

### ISSUE THREAD

**Issue by ThinkChaos** [pietheinstrengholt/rssmonster#26]
Right now because the configurations are in the `client` and `server` directories, any change requires rebuilding the docker image. It would be nice to not have to do so.

Possible solutions:
 - move the config to another direcory
 - better use of docker (I don't know if this is possible without running into the same bug as #22)

**Comment by pietheinstrengholt** [comment]
A solution can be is to create two docker images. One for the front-end, which serves the client via NGINX and another one for the back-end, which serves out the server directory.

**Comment by pietheinstrengholt** [comment]
I have completely restructured the way docker has been used. The client and server now have their own running instances. Both the volumes are mapped. This means if you make a change to the source code it is seen by the docker instance, so both vuejs and node will do a hot refresh. The client consequently now also serves out the data (instead of building and moving it to the server folder).
Btw, if you use WORKDIR and the directory doesn't exist, it will be automatically created on the fly.
I also added a dockerignore file, because we are creating the node_modules folder inside the container. Therefore it makes no sense to add it to the folder. We better ignore it.
Lastly I renamed the app to server and added an additional client file. This better matches the structure of the project.
